cms/envs/common.py

```diff
@@ -461,13 +459,6 @@
 
 ################################# Middleware ###################################
 
-# TODO: Remove Django 1.11 upgrade shim
```

Comment:
I think you can remove the birdcage requirement along with this, since I'm fairly sure that this is all it does.

Reply:
Good point. Removing and running make up, hence the other unrelated plugin bumps.